policyengine_us/variables/gov/states/nm/tax/income/exemptions/nm_blind_and_aged_exemption.py

```diff
@@ -0,0 +1,62 @@
+from policyengine_us.model_api import *
+
+
+class nm_aged_blind_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Mexico aged and blind exemption"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        # 7-2-5.2. EXEMPTION--INCOME OF PERSONS SIXTY-FIVE AND OLDER OR BLIND
+        "https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/856ebf4b-3814-49dd-8631-ebe579d6a42b/Personal%20Income%20Tax.pdf",
+        # Tax Form Instructions Page ADJ-5 TABLE 1. Exemptions for Persons 65 or Older or Blind
+        "https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/1afc56af-ea90-4d48-82e5-1f9aeb43255a/PITbook2022.pdf",
+    )
+    defined_for = StateCode.NM
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.nm.tax.income.exemptions.blind_and_aged
+        filing_status = tax_unit("filing_status", period)
+        statuses = filing_status.possible_values
+        agi = tax_unit("adjusted_gross_income", period)
+        age_threshold = p.age_threshold
+
+        # check if taxpayer is blind or aged
+        blind_head = tax_unit("blind_head", period).astype(int)
+        aged_head = (tax_unit("age_head", period) >= age_threshold).astype(int)
+
+        # check if taxpayer is eligible (Taxpayers cannot take exemptions for being both 65 or older and blind.)
+        head_eligible = blind_head | aged_head
+
+        # check if spouse is blind or aged
+        blind_spouse = tax_unit("blind_spouse", period).astype(int)
```

Comment:
Don't need to cast any of these to ints because you're converting to a bool with `|` later.
```suggestion
        blind_spouse = tax_unit("blind_spouse", period)
```